Makefile

```diff
@@ -1,6 +1,6 @@
 ADB = ${ANDROID_HOME}/platform-tools/adb
 GRADLEW = ./gradlew
-flavour = UnbrandedWebview
+flavour = icm_ph_chc
```

Comment:
No need to replace the default flavor to build another.

Reply:
Thanks, I have reverted the change.